core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheFileInStream.java

```diff
@@ -125,6 +132,7 @@ private int readInternal(byte[] b, int off, int len, ReadType readType, long pos
     int totalBytesRead = 0;
     long currentPosition = pos;
     long lengthToRead = Math.min(len, mStatus.getLength() - pos);
+    Stopwatch readTimeStopwatch = Stopwatch.createUnstarted(mTicker);
```

Comment:
do we need to create one `Stopwatch` per read?
possibly to share one within a `LocalCacheFileInStream`?

Reply:
Done, made stopwatch as a field in LocalCacheFileInStream